source/basic.tex

```diff
@@ -3899,6 +3899,8 @@
 in~\ref{conv.prom}.}
 A synonym for integral type is
 \indextext{integer~type}%
+\indextext{complement!ones'}%
+\indextext{complement!two's}%
```

Comment:
Should the index entry not be "complex" then? Maybe change it to "Signed integer representation", and have sub-entries "Ones' complement", "Two's complement", and "Signed magnitude"?

Are there other places where "signed magnitude" should be indexed, or other representations that should be indexed?